src/client/ProductClient.ts

```diff
@@ -0,0 +1,34 @@
+import { APIConstants } from './../constants/APIConstants';
+import { BaseClient } from './BaseClient';
+import { ProductsData, ProductData } from '../types/Product';
+
+export class ProductClient extends BaseClient {
+  /**
+   * @ignore
+   */
+  public constructor(private readonly accessToken: string) {
+    super();
+  }
+
+  /**
+   *
+   * @param id Collection ID
+   */
+  async getAllByCollectionId(id: string): Promise<ProductsData> {
```

Comment:
```suggestion
  async getAllByCollectionId(collectionId: string): Promise<ProductsData> {
```

You can use the variable name `id` only when you are referring to entity ID in entity service / controller / client. Here since we are in product client, only product ID should be termed as `id`. Other ids should be explicitly mentioned

Reply:
Okay, i uderstood